osgi-dependencies/proxy/src/main/java/com/cognifide/aet/proxy/ProxyServerProvider.java

```diff
@@ -44,19 +44,20 @@
           unbind = "unbindProxyManager")
   private final Map<String, ProxyManager> collectorManagers = Maps.newConcurrentMap();
 
-  public ProxyServerWrapper createProxy(String useProxy, int port) throws ProxyException {
-    String proxyType = useProxy;
-    if ("true".equals(useProxy)) {
+  public ProxyServerWrapper createProxy(String usedProxy) throws ProxyException {
+    String proxyType = usedProxy;
+    if ("true".equals(usedProxy)) {
```

Comment:
Thank you @toniedzwiedz . I have removed the changes related to webProxy config.
They should be consulted and addressed in another pull request in future.